lib/camping/commands.rb

```diff
@@ -111,5 +362,41 @@ def self.routes(theApp = Camping, silent = false)
       return routes
     end
 
+    def self.new_cmd(app_name=:Camp)
+      app_name = :Camp if app_name == nil
+      app_name = app_name.to_sym if app_name.class == String
+
+      snake_app_name = Camping::CommandsHelpers.to_snake_case(app_name)
+
+      # make a directory then move there.
+      # _original_dir = Dir.pwd
+      Dir.mkdir("#{snake_app_name}") unless Dir.exist?("#{snake_app_name}")
+      Dir.chdir("#{snake_app_name}")
+
+      # generate a new camping app in a directory named after it:
+      Generators::make_camp_file(app_name)
+      Generators::make_gitignore()
+      # make_special_rakefile()
+      Generators::make_rakefile()
+      Generators::make_ruby_version()
+      Generators::make_configkdl()
+      Generators::make_gemfile()
+      Generators::make_readme()
+      Generators::make_public_folder()
+      Generators::make_test_folder()
+
+      # optionally add omnibus support
+        # add src/ folder
+        # add lib/ folder
+        # add views/ folder
+
+      # optionally add a local database too, through guidebook
+        # add db/ folder
+        # add db/migrate folder
+        # add db/config.kdl
+        # append migrations stuff to Rakefile.
+        `ls`
```

Comment:
Is this to run after all the commands have generated the files/folders?

Reply:
Yes, to kinda display, shallowly, the files created by the action. It's a very lazy, and naive way to do this. We can do something better when we improve the Generators again.